src/cursor.c

```diff
@@ -2103,11 +2103,16 @@ if (0) fprintf(stderr, "check_x11_pointer %d %d\n", root_x, root_y);
 		}
 	}
 
+	int x_rotated = x, y_rotated = y;
+	if (screen) {
+		rotate_cursor_coords(x, y, &x_rotated, &y_rotated, screen->width, screen->height);
```

Comment:
This links rather by chance as it's not explicitly exported. Why don't you use `rotate_coords()` from scan.h, which is already there?

Reply:
Does just adding the function to the header file fix that? 
I haven't done C in a long time, I'm a little lost :smile:

Please check the latest version; i added the function header there; if that doesn't fix the issue, please tell me explicitly what to change.